app/controllers/CompanyDetailsController.scala

```diff
@@ -40,7 +41,7 @@ class CompanyDetailsController @Inject()(
                                           cc: MessagesControllerComponents,
                                           view: CompanyDetailsView
                                         )(implicit executionContext: ExecutionContext)
-  extends FrontendController(cc) with I18nSupport {
+  extends FrontendController(cc) with I18nSupport with WithDefaultFormBinding{
```

Comment:
From some testing I don't think the `with WithDefaultFormBinding` isn't required as the form binds fine without it?
As per the [confluence post](https://confluence.tools.tax.service.gov.uk/pages/viewpage.action?pageId=447906456#:~:text=%C2%A0%C2%A0with%20WithDefaultFormBinding%20%7B-,WithDefaultFormBinding,-allows%20you%20to) we don't want to add this unless absolutely necessary as it re-introduces the vulnerability.

Reply:
Without this the tests failed for me thats why I added it.